topi/python/topi/cuda/conv2d_winograd.py

```diff
@@ -42,25 +45,24 @@ def winograd_cuda(cfg, data, kernel, strides, padding, dilation, layout, out_dty
 
     N, CI, H, W = get_const_tuple(data.shape)
 
+    if isinstance(dilation, int):
+        dilation_h = dilation_w = dilation
+    else:
+        dilation_h, dilation_w = dilation
+    HPAD, WPAD, _, _ = nn.get_pad_tuple(padding, kernel)
```

Comment:
does this work when kernel is pre-computed?

Reply:
Thanks for pointing out! I moved it after the kernel transformation.